Makefile

```diff
@@ -119,6 +119,25 @@ clean: .cleanpycs .cleandocs
 	@echo "Removing generated documentation"
 	rm -rf $(DOC_BUILD_DIR)
 
+.PHONY: .cleanmongodb
+.cleanmongodb:
+	@echo "==================== cleanmongodb ===================="
+	@echo "----- Dropping all MongoDB databases -----"
+	@sudo service mongodb force-stop
+	@sudo rm -rf /var/lib/mongodb/*
+	@sudo chown -R mongodb:mongodb /var/lib/mongodb/
+	@sudo service mongodb start
+	@mongo --eval "rs.initiate()"
```

Comment:
You still need to run initiate after you nuke the db folder, even though we don't use replication (otherwise the server thinks it's a slave).